app/src/main/java/com/duckduckgo/app/di/AppComponent.kt

```diff
@@ -79,7 +80,8 @@ import javax.inject.Singleton
         CertificateTrustedStoreModule::class,
         WelcomePageModule::class,
         HttpsPersisterModule::class,
-        EmailModule::class
+        EmailModule::class,
+        BookmarksModule::class,
```

Comment:
I think we should start using `ContributesTo(AppObjectGraph::class)` in the module.